diff --git a/build/index.d.ts b/build/index.d.ts
index 2fb824c961c530ac1892f87e4292b3012bd773f9..0d8ca467b4d7ec0f01532623dcf15d782f8c34e1 100644
--- a/build/index.d.ts
+++ b/build/index.d.ts
@@ -9,14 +9,18 @@ import {TransformOptions} from '@babel/core';
 import {SyncTransformer, TransformerCreator} from '@jest/transform';
 
 export declare const createTransformer: TransformerCreator<
-  SyncTransformer<TransformOptions>,
-  TransformOptions
+  SyncTransformer<TransformerConfig>,
+  TransformerConfig
 >;
 
+declare interface TransformerConfig extends TransformOptions {
+  excludeJestPreset?: boolean;
+}
+
 declare const transformerFactory: {
   createTransformer: TransformerCreator<
-    SyncTransformer<TransformOptions>,
-    TransformOptions
+    SyncTransformer<TransformerConfig>,
+    TransformerConfig
   >;
 };
 export default transformerFactory;
diff --git a/build/index.js b/build/index.js
index c3b081b2567ee17e2f23c2d0104c7ae671e8189f..1cc3f4c7b8a8896eb1eef6ac30582d31cf53d801 100644
--- a/build/index.js
+++ b/build/index.js
@@ -68,7 +68,7 @@ function _core() {
 /******/ 	
 /************************************************************************/
 var __webpack_exports__ = {};
-// This entry need to be wrapped in an IIFE because it need to be isolated against other modules in the chunk.
+// This entry need to be wrapped in an IIFE because it uses a non-standard name for the exports (exports).
 (() => {
 var exports = __webpack_exports__;
 
@@ -120,7 +120,7 @@ function _slash() {
   return data;
 }
 var _loadBabelConfig = __webpack_require__("./src/loadBabelConfig.ts");
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+function _interopRequireDefault(e) { return e && e.__esModule ? e : { default: e }; }
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
 /**
@@ -185,8 +185,11 @@ async function loadBabelOptionsAsync(cwd, filename, transformOptions, jestTransf
   } = await loadBabelConfigAsync(cwd, filename, transformOptions);
   return addIstanbulInstrumentation(options, jestTransformOptions);
 }
-const createTransformer = userOptions => {
-  const inputOptions = userOptions ?? {};
+const createTransformer = transformerConfig => {
+  const {
+    excludeJestPreset,
+    ...inputOptions
+  } = transformerConfig ?? {};
   const options = {
     ...inputOptions,
     caller: {
@@ -199,7 +202,7 @@ const createTransformer = userOptions => {
     },
     compact: false,
     plugins: inputOptions.plugins ?? [],
-    presets: [...(inputOptions.presets ?? []), jestPresetPath],
+    presets: [...(inputOptions.presets ?? []), ...(excludeJestPreset === true ? [] : [jestPresetPath])],
     sourceMaps: 'both'
   };
   function mergeBabelTransformOptions(filename, transformOptions) {
